src/coreclr/gc/gc.cpp

```diff
@@ -11432,7 +11442,7 @@ void gc_heap::init_heap_segment (heap_segment* seg, gc_heap* hp
     heap_segment_next (seg) = 0;
     heap_segment_plan_allocated (seg) = heap_segment_mem (seg);
     heap_segment_allocated (seg) = heap_segment_mem (seg);
-    heap_segment_saved_allocated (seg) = heap_segment_mem (seg);
+    save_allocated(seg);
```

Comment:
>   save_allocated(seg) [](http://example.com/codeflow?start=1&length=22)

we don't need to change this as we are just initializing. we are very intentional about initializing saved_allocated to mem.